packages/bot/src/stores/chart-store.js

```diff
@@ -0,0 +1,82 @@
+
+import { action,
+    computed,
+    observable }  from 'mobx';
+import ServerTime from '../services/api/server_time';
+
+const g_subscribers_map = {};
```

Comment:
is `g` means `granularity` here?

Reply:
@mustofa-binary this i copied from trader but i believe it stands for group. not sure but I want to keep it like trader